base/c.jl

```diff
@@ -4,6 +4,11 @@
 
 import Core.Intrinsics: cglobal, bitcast
 
+# Equivalent to Sys.iswindows(), but that's not yet defined
+const _WINDOWS = let k = ccall(:jl_get_UNAME, Any, ())
```

Comment:
giving it a name means it's there in Base for everyone to misuse. the ccall happens only at system image build time. otherwise could use a let to avoid occupying namespace in Base